numerics/fast_sin_cos_2π.hpp

```diff
@@ -0,0 +1,13 @@
+﻿
+#pragma once
+
+namespace principia {
+namespace numerics {
+
+// An implementation of sin and cos that favours performance at the expense of
+// accuracy.  It has an absolute error of roughly 6e-7 and takes about 25 CPU
+// cycles.
```

Comment:
Benchmark with 64-bit argument reduction, if that's measurably slower, document the range limitation.

Consider DCHECKing and returning NaN outside the correct range (returning NaN when out of range should be free with branch prediction).

Reply:
OBE.